### ISSUE TITLE
Use of `get` method on dictionaries

### ISSUE THREAD

**Issue by harleylara** [EOLab-HSRW/jetson-models#1]
DO NOT use the `get` method on dictionaries. The `get` method DO NOT raise a `KeyError` **if the given key non-existent** ([python docs](https://docs.python.org/2/library/stdtypes.html#dict.get)). Please use square bracket syntax to get values from dictionaries.

```python
demo = {"test": "hi"}
demo.get("noakey") # this DOES NOT throw error !!
demo["noakey"] # this does
```

**Comment by harleylara** [comment]
Please add as a comment the commit or set of commit that fix this issue. 😁

side node: use more descriptive commit message during commit avoid using the same `"Updated models"`.

**Comment by DCenten0** [comment]
I have fixed this issue in: 015c61f
In this update, I replaced `data.get('key')` with `data['key']` to improve value extraction from dictionaries and ensure `KeyError` are raised when keys are missing or wrong.

And sorry I did not know how to link the commit with the issue 😅 and thank you next commits going to be clearer
